structuredefinitions/Extension-UKCore-ConditionEpisode.xml

```diff
@@ -53,6 +53,10 @@
       <type>
         <code value="CodeableConcept" />
       </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="https://fhir.hl7.org.uk/ValueSet/UKCore-ConditionEpisodity" />
```

Comment:
Should be: https://fhir.hl7.org.uk/ValueSet/UKCore-ConditionEpisodicity

Reply:
Also is it right to just be using https://fhir.hl7.org.uk/ValueSet/UKCore-ConditionEpisodicity? I assume ConditionEpisode isn't also included because it's retired.